config/mongo.js

```diff
@@ -1,36 +1,29 @@
 import mongoose from "mongoose";
-import * as fs from "fs";
+import { env } from "process";
+import logger from "./logger";
 
-import logger from "../config/logger";
+let connection;
 
-let hasConnection;
 const connectMongo = async () => {
-  if (!process.env.LINKFREE_MONGO_CONNECTION_STRING) {
+  const { LINKFREE_MONGO_CONNECTION_STRING } = process.env;
+
+  if (!LINKFREE_MONGO_CONNECTION_STRING) {
     throw new Error(
       "Please define the LINKFREE_MONGO_CONNECTION_STRING environment variable (if local add to .env file)"
     );
   }
 
-  if (hasConnection) {
-    return hasConnection;
+  if (connection) {
+    return connection;
   }
-  try {
-    // DigitalOcean Apps has cert as environment variable but Mongo needs a file path
-    // Write Mongo cert file to disk
-    if (process.env.CA_CERT) {
-      fs.writeFileSync("cert.pem", process.env.CA_CERT);
-    }
 
-    hasConnection = await mongoose.connect(
-      process.env.LINKFREE_MONGO_CONNECTION_STRING,
-      { autoIndex: true, family: 4 }
-    );
-    hasConnection = true;
-    logger.info("MongoDB connected");
-    return hasConnection;
-  } catch (e) {
-    hasConnection = false;
-    logger.error(e, "DB connection failed");
+  try {
+    const options = { autoIndex: true, family: 4 };
+    connection = await mongoose.connect(LINKFREE_MONGO_CONNECTION_STRING,options);
+    logger.info("MongoDB connected:", connection.name);
```

Comment:
Hey 👋 we can also improve this log to something like 
`DB connection successful:` to keep it in sync with the error log in line 25.